manage/groups/settings.rst

```diff
@@ -3,40 +3,104 @@ Group Settings
+.. figure:: /images/manage/groups/group-management.png
+   :alt: Screenshot showing Zammads group management
 
-- *Name*: choose a name for the group.
+No matter if you're going to edit or create a new group, each group
+comes with the following settings you can adjust as needed.
 
-- *Assignment timeout*: the time in minutes after which the ticket's state will
-  revert back to *unassigned* after the assigned agent hasn't worked on the
-  ticket.
+Name
+   This is the name your agents (and customers when using :doc:`/channels/web`)
+   will see within the Zammad-UI.
 
-- *Follow up possible*: configure what happens when a customer replies to a
-  closed ticket:
+Assignment timeout
+   The time in minutes after which the ticket's ownership will revert back to
+   *unassigned* after the assigned agent hasn't worked on the ticket.
 
-  - *yes*: the ticket will be reopened.
-  - *do not reopen ticket but create new Ticket*: the ticket will remain closed,
-     and Zammad will instead create a new ticket for the customer's reply.
+      .. note::
 
-- *Assign follow ups*: configure whether a closed ticket that has been reopened
-  due to a customer's reply should remain assigned to the last agent:
+         This timeout does not take any working hours in account.
 
-  - *yes*: the ticket will remain to the last agent who owned it.
-  - *no*: Zammad will unassign the ticket.
+Follow-up possible
+   This option allows you to decide how Zammad should react if a customer
+   replies to a closed ticket (no matter if by e.g. email or UI).
 
-- *Email*: choose which email address will be used as the sending address
-  (`From` header) when agents reply to tickets in this group. Email addresses
-  can be configured in Channels → Email → Accounts.
+   yes
+      The ticket will be reopened. This is the default value.
+   
+   do not reopen Ticket but create new Ticket
+      The ticket will remain closed and Zammad will create a new ticket instead.
+      The new ticket contains the customers reply only.
 
-- *Signature*: choose which :doc:`signature </channels/email/signatures>` to use
-  when replying to tickets in this group.
+Assign follow-ups
+   This setting allows you to decide if, upon a reopen of a ticket, the last
+   assigned owner should stay assigned or if Zammad should reset the owner to
+   nobody.
 
-- *Note*: an internal note about the group that is only visible to people who
-  can access the group management area.
+   yes
+      The ticket will remain to the last agent who owned it.
+      This is the default value
+   no
+      The owner assignment of the ticket will be removed.
 
-- *Active*: choose whether the group is enabled or not. Groups cannot be
-  deleted, they can only be set to inactive.
+Email
+   Ticket groups decide on which email address Zammad will use outbound
+   for replying on a ticket. *Destination groups* of your email based channels
+   do not have any effect on this setting and may differ.
```

Comment:
I like the former wording for the Email field much better. I would skip the comment about destination groups, I think the number of people that may be confused by it is higher than the number of people that are helped with it at this point. Instead I would add a comment to the documentation of destination groups, that the sender is configured in the group settings.

Reply:
Fun fact: There's already such comments in the email section and it's in general overseen.
This is why there's an enhancement issue for this on this repo.

Do you still think having it twice is a bad idea?

Reply:
Haven't checked the documentation of the Email-section to be honest. Great to see, that there is a comment about the senders address already.

But yes, still prefer the old wording here.

Reply:
So you want only the following?


```
Email
   Ticket groups decide on which email address Zammad will use outbound
   for replying on a ticket.

      .. note:: **You don't have an email address configured yet?**

         Please configure an email based channel before here

            * :doc:`Channels → Email </channels/email/index>`
            * :doc:`Channels → Microsoft 365 </channels/microsoft365/index>`
            * :doc:`Channels → Google </channels/google/index>`

         and come back afterwards.
 ```

Reply:
Suggestion:
Select which sender's email address Zammad will use outbound for replying on a ticket in this group.

Instead of:
Ticket groups decide on which email address Zammad will use outbound
   for replying on a ticket. *Destination groups* of your email based channels
   do not have any effect on this setting and may differ.